README.md

```diff
@@ -46,9 +49,6 @@ If you for any reason find yourself behind a transparent proxy you need to set t
 - Set the environment variables in your current session by running `export SSL_CERT_FILE=<path/to/ca-certificates-file>` and `export CURL_CA_BUNDLE=<path/to/ca-certificates-file>` in the terminal. Eg:`export SSL_CERT_FILE=/etc/ssl/certs/ca-certificates.crt CURL_CA_BUNDLE=/etc/ssl/certs/ca-certificates.crt`
 - Set the environment variables permanently by adding the export commands above to your `~/.bashrc` or equivalent.
 
-### Extending the base-box to fit your needs
-To see a full example of how to utilise this box and extend it, go to [template-repo](https://github.com/fredrikhgrelland/vagrant-hashistack-template).
-
```

Comment:
Didn't see this before approving the first time, but why would you like to remove this?

Reply:
Moved it up and reworded it a bit. Thought that plain and templated should be together.

Reply:
My bad, I didn't read `Starter template` section well enough.